modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/KaleoDesigner.macro

```diff
@@ -285,6 +285,20 @@ definition {
 		KaleoDesigner.viewNodeNotPresent(nodeNodeLabel = ${nodeNodeLabel});
 	}
 
+	macro deleteNodeInKaleoForms {
+		var key_nodeType = ${nodeNodeLabel};
```

Comment:
The tests described in the ticket were failing due to the macro that is used (deleteNode) only in the Worklow that is in the Process Builder. Visualizing the Workflow in Kaleo Forms is totally different.